_drafts/gallery.md

```diff
@@ -28,7 +28,7 @@ image:
 #
 # Metainformation & Customization
 #
-meta_description:
+description:
```

Comment:
The Jekyll SEO gem [uses `description`](https://github.com/jekyll/jekyll-seo-tag/#usage) to output description meta.